pages/getting-started/install-memgraph/amazon-linux.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Install Memgraph on Amazon Linux from RPM package
+description: Install Memgraph from RPM package on Amazon Linux simply and efficiently using our documentation. 
+---
+
+import { Callout } from 'nextra/components'
+import { Steps } from 'nextra/components'
+import CodeSnippet from '/components/code-snippet/CodeSnippet'
+
+# Install Memgraph on Amazon linux from RPM package
+
+Install and run Memgraph from a RPM package on: 
+  - **Amazon Linux 2**
+
+  <Callout type="warning"> 
+Newer versions of Memgraph might not be backward
+compatible with older versions and you might not be able to load storage
+snapshots between different versions. In that case, please contact the Memgraph
+team and we'll help with the migration as much as possible.
+ </Callout>
+
+  ## Installation guide
+
+  <Steps>
+
+  {<h3>Download Memgraph RPM Package</h3>}
+
+Downloaded the latest Memgraph RPM Package from the [Memgraph Download
+Hub](https://memgraph.com/download/).
+
+You can also use [direct download
+links](/getting-started/install-memgraph/direct-download-links) to get the
+latest Memgraph packages. 
+
+For example for Amazon Linux 2:
+
+<CodeSnippet code="sudo wget https://download.memgraph.com/memgraph/v2.15.1/amzn-2/memgraph-2.15.1_1-1.x86_64.rpm" page="install-memgraph/amazon-linux" os="amazon-linux"/>
+
+{<h3>Install Memgraph</h3>}
+
```

Comment:
Please add note like on https://memgraph.com/docs/getting-started/install-memgraph/ubuntu#installation-guide (we had users reporting this):
For the latest version of Memgraph, replace the version in the download command with the one from the [direct download links](https://memgraph.com/docs/getting-started/install-memgraph/direct-download-links) page.

Reply:
same goes for other OS

Reply:
This has been added to the latest commit.